src/Relude/Foldable/Fold.hs

```diff
@@ -73,8 +72,8 @@ flipfoldl' f = foldl' (flip f)
 >>> asumMap (\x -> if x > 2 then Just x else Nothing) [1..4]
 Just 3
 -}
-asumMap :: (Foldable f, Alternative m) => (a -> m b) -> f a -> m b
-asumMap f = getAlt . foldMap (Alt . f)
+asumMap :: forall m b f a . (Foldable f, Alternative m) => (a -> m b) -> f a -> m b
```

Comment:
Now that we changed `forall` arguments order, I wonder if we should update it to be consistent with `foldMapA`. So, I propose either to change it to `b m f a` or to change `foldmapA` function to have the argument in the same order as here. Let me know what you both think @chshersh @josephcsible .

Reply:
@vrom911 good point about consistent variable order :+1: I missed that part. Order of variables in `foldMapA` is optimal. So let's instead change the order in `asumMap` to the same as in `foldMapA`. Just to have less API surprises.

Reply:
Done.